optimum/intel/openvino/quantization.py

```diff
@@ -508,17 +508,23 @@ def build_from_dataset(
             elif isinstance(self.model, _OVModelForWhisper):
                 return self._prepare_speech_to_text_calibration_data(quantization_config, dataset)
             elif isinstance(self.model, OVModelForSeq2SeqLM):
-                return self._prepare_text_to_text_calibration_data(quantization_config, dataset)
+                return self._prepare_text_to_text_calibration_data(
+                    quantization_config, dataset, **quantization_config.dataset_kwargs
```

Comment:
```suggestion
                    quantization_config, dataset, seq_len=config.dataset_kwargs.get("seq_len")
```

Let's do this way here and in similar places below.